lib/msal-node/src/cache/Storage.ts

```diff
@@ -240,15 +240,14 @@ export class Storage extends CacheManager {
      */
     getKeys(): string[] {
         // read inMemoryCache
-        const cache = this.getCache();
+        const cache: InMemoryCache= this.getCache() as InMemoryCache;
         let cacheKeys: string[] = [];
 
-        // read all keys
-        Object.keys(cache).forEach(key => {
-            Object.keys(key).forEach(internalKey => {
-                cacheKeys.push(internalKey);
-            });
-        });
+        cacheKeys = Object.keys(cache.accounts) as Array<string>;
+        cacheKeys.push(...Object.keys(cache.idTokens) as Array<string>);
+        cacheKeys.push(...Object.keys(cache.accessTokens) as Array<string>);
+        cacheKeys.push(...Object.keys(cache.refreshTokens) as Array<string>);
+        cacheKeys.push(...Object.keys(cache.appMetadata) as Array<string>);
```

Comment:
```js
return [ 
  ...Object.keys(cache.accounts),
  ...Object.keys(cache.idTokens),
  ...Object.keys(cache.accessTokens),
  ...Object.keys(cache.refreshTokens),
  ...Object.keys(cache.appMetadata)
];

Reply:
Addressed this. Thanks, this looks much better.